pkg/util/metrics_helper_test.go

```diff
@@ -197,6 +197,78 @@ func TestSendMaxOfGauges(t *testing.T) {
 	require.ElementsMatch(t, expected, actual)
 }
 
+func TestSendSumOfHistogramsWithLabels(t *testing.T) {
```

Comment:
As this test function contains multiple tests, have you considered using blocks to separate them more visibly in the code, and to avoid mixing variables between them?

Reply:
Good point, done.